.orijtech/bencher.yaml

```diff
@@ -0,0 +1 @@
+haha im malformed what're you gonna do about it
```

Comment:
Wrong file name .bencher/config.yaml is what you want.

Reply:
Haha yeah, and to think I wrote documentation for this just yesterday 🙃